aptos-node/src/lib.rs

```diff
@@ -127,84 +127,83 @@ pub fn load_test_environment<R>(
 ) where
     R: ::rand::RngCore + ::rand::CryptoRng,
 {
-    // Either allocate a temppath or reuse the passed in path and make sure the directory exists
     let config_temp_path = aptos_temppath::TempPath::new();
-    let config_path = config_path.unwrap_or_else(|| config_temp_path.as_ref().to_path_buf());
+
+    let (try_load, config_path) = if let Some(config_path) = config_path {
+        (config_path.join("0/node.yaml").exists(), config_path)
+    } else {
+        (false, config_temp_path.as_ref().to_path_buf())
+    };
+
     std::fs::DirBuilder::new()
         .recursive(true)
         .create(&config_path)
         .unwrap();
+
     let config_path = config_path.canonicalize().unwrap();
 
-    // Build a single validator network
-    let mut maybe_config = PathBuf::from(&config_path);
-    maybe_config.push("validator_node_template.yaml");
-    let mut template = NodeConfig::load_config(maybe_config)
-        .unwrap_or_else(|_| NodeConfig::default_for_validator());
+    let validator_config_path = config_path.join("0/node.yaml");
```

Comment:
nit: this is not windows friendly

Reply:
hmm.... I dream of the day where a doc is clear whether or not path separators are universal or need to be managed by the programmer...